spec/services/service_provider_seeder_spec.rb

```diff
@@ -142,6 +139,15 @@
       context 'in another environment' do
         let(:deploy_env) { 'int' }
 
+        it 'only writes configs with restrict_to_deploy_env for sandbox' do
+          run
+
+          expect(ServiceProvider.find_by(issuer: sandbox_issuer)).to be_present
```

Comment:
**suggestion:** You could update to also test that it writes to another "sandbox"-category environment (dev).

Reply:
I just noticed you're testing that below with `unrestricted_issuer`, so not a big deal.